boards/steelseries/apexpromini/apexpromini.dts

```diff
@@ -0,0 +1,249 @@
+			label = "User dbg tp";
+		};
+	};
+
+	aliases {
+		led0 = &dbg_led;
+		volt-sensor0 = &vref;
+		volt-sensor1 = &vbat;
+	};
+
+	kbd: kbd-matrix {
+		compatible = "gpio-kbd-matrix";
+		status = "okay";
+		idle-mode = "poll";
+		settle-time-us = <110>;
+		no-ghostkey-check;
+		col-drive-inactive;
+		col-gpios
+			= <&gpiob 0  (GPIO_ACTIVE_HIGH | GPIO_PULL_DOWN)>
+			, <&gpiob 1  (GPIO_ACTIVE_HIGH | GPIO_PULL_DOWN)>
+			, <&gpiob 2  (GPIO_ACTIVE_HIGH | GPIO_PULL_DOWN)>
+			, <&gpiob 10 (GPIO_ACTIVE_HIGH | GPIO_PULL_DOWN)>
+			, <&gpiob 11 (GPIO_ACTIVE_HIGH | GPIO_PULL_DOWN)>
+			;
+			row-gpios
+			= <&gpioa 0  GPIO_ACTIVE_HIGH>
+			, <&gpioa 1  GPIO_ACTIVE_HIGH>
+			, <&gpioa 2  GPIO_ACTIVE_HIGH>
+			, <&gpioa 3  GPIO_ACTIVE_HIGH>
+			, <&gpioa 4  GPIO_ACTIVE_HIGH>
+			, <&gpioa 5  GPIO_ACTIVE_HIGH>
+			, <&gpioa 6  GPIO_ACTIVE_HIGH>
+			, <&gpioa 7  GPIO_ACTIVE_HIGH>
+			, <&gpioc 4  GPIO_ACTIVE_HIGH>
+			, <&gpioc 5  GPIO_ACTIVE_HIGH>
+			, <&gpioc 0  GPIO_ACTIVE_HIGH>
+			, <&gpioc 1  GPIO_ACTIVE_HIGH>
+			, <&gpioc 2  GPIO_ACTIVE_HIGH>
+			, <&gpioc 3  GPIO_ACTIVE_HIGH>
+			;
+	};
+};
+
+&clk_lsi {
+	status = "okay";
+};
+
+&clk_hsi {
+	status = "okay";
+};
+
+&clk_hsi48 {
+	status = "okay";
+};
+
+&pll {
+	div-m = <1>;
+	mul-n = <10>;
+	div-q = <2>;
+	div-r = <2>;
+	clocks = <&clk_hsi>;
+	status = "okay";
+};
+
+&rcc {
+	clocks = <&pll>;
+	clock-frequency = <DT_FREQ_M(80)>;
+	ahb-prescaler = <1>;
+	apb1-prescaler = <1>;
+	apb2-prescaler = <1>;
+};
+
+&usart1 {
+	pinctrl-0 = <&usart1_tx_pb6 &usart1_rx_pb7>;
+	pinctrl-names = "default";
+	current-speed = <115200>;
+	status = "okay";
+};
+
+&spi1 {
+	pinctrl-0 = <&spi1_sck_pb3 &spi1_miso_pb4 &spi1_mosi_pb5>;
+	pinctrl-names = "default";
+	cs-gpios = <&gpioa 15 (GPIO_ACTIVE_LOW | GPIO_PULL_UP)>;
+	status = "okay";
+
+	flash1: fm25q128c@0 {
+		compatible = "jedec,spi-nor";
+		reg = <0>;
+		spi-max-frequency = <10000000>;
+		size = <0xfa0000>;
+		jedec-id = [a1 40 14];
+	};
+};
+
+&spi2 {
+	pinctrl-0 = <&spi2_sck_pb13 &spi2_miso_pb14 &spi2_mosi_pb15>;
+	pinctrl-names = "default";
+	cs-gpios = <&gpiob 12 (GPIO_ACTIVE_LOW | GPIO_PULL_UP)>;
+	status = "okay";
+};
+
+&timers2 {
+	status = "okay";
+
+	pwm2: pwm {
+		status = "okay";
+		pinctrl-0 = <&tim2_ch1_pa0>;
+		pinctrl-names = "default";
+	};
+};
+
+&rtc {
+	clocks = <&rcc STM32_CLOCK_BUS_APB1 0x10000000>,
+		 <&rcc STM32_SRC_LSI RTC_SEL(2)>;
+	status = "okay";
+};
+
+&flash1 {
+	partitions {
+		compatible = "fixed-partitions";
+		#address-cells = <1>;
+		#size-cells = <1>;
+		storage_partition: partition@0 {
+			label = "storage";
+			reg = <0x0 DT_SIZE_K(16384)>;
+		};
+	};
+};
+
+&usb_fs_phy {
+	status = "okay";
+};
+
+zephyr_udc0: &usb {
+	status = "okay";
+	pinctrl-0 = <&usb_dm_pa11 &usb_dp_pa12 &usb_noe_pa13>;
+	pinctrl-names = "default";
+
+	cdc_acm_uart0: cdc_acm_uart0 {
+		compatible = "zephyr,cdc-acm-uart";
+	};
+};
+
+&vref {
+	status = "okay";
+};
+
+&vbat {
+	status = "okay";
+};
+
+&kbd {
+	keymap {
```

Comment:
you can put this straight under the kbd node on line 69 if you want to, no point shuffling nodes around